subprojects/wrapper/src/main/java/org/gradle/wrapper/GradleWrapperMain.java

```diff
@@ -66,9 +66,10 @@ public static void main(String[] args) throws Exception {
                 new BootstrapMainStarter());
     }
 
-    private static void addSystemProperties(File gradleHome, File rootDir) {
-        System.getProperties().putAll(SystemPropertiesHandler.getSystemProperties(new File(gradleHome, "gradle.properties")));
-        System.getProperties().putAll(SystemPropertiesHandler.getSystemProperties(new File(rootDir, "gradle.properties")));
+    private static void addSystemProperties(Properties systemProperties, File gradleHome, File rootDir) {
```

Comment:
Should be called `gradleUserHome`

Reply:
Probably. Maybe. But that's out of scope of this change, as the variable was called like that before.

Reply:
I've anyway added a new commit doing the rename.